lib/Ed25519VerificationKey2020.js

```diff
@@ -69,14 +91,24 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
    * @returns {Ed25519VerificationKey2020} - 2020 suite instance.
    */
   static fromEd25519VerificationKey2018({keyPair} = {}) {
+    const publicKeyMultibase = MULTIBASE_BASE58BTC_HEADER +
+      base58btc.encode(new Uint8Array([
+        ...MULTICODEC_ED25519_PUB_HEADER,
+        ...base58btc.decode(keyPair.publicKeyBase58)
+      ]));
```

Comment:
I know this looks so very pretty, but the spread operator is quite slow for this (yes, it's been tested).

```suggestion
      const key = base58btc.decode(keyPair.publicKeyBase58);
      const mk = new Uint8Array(
        MULTICODEC_ED25519_PUB_HEADER.length + key.length);
      mk.set(MULTICODEC_ED25519_PUB_HEADER);
      mk.set(key, MULTICODEC_ED25519_PUB_HEADER.length);
      const publicKeyMultibase = MULTIBASE_BASE58BTC_HEADER +
        base58btc.encode(mk);
```

Reply:
Fixed in ce7ff1158384b8bf1c65998676bfc5ac3edd604a.